src/LifeCycle.hs

```diff
@@ -96,7 +96,9 @@ phaseTrans _ _ = Nothing
 data Capability
     -- Idea
     = CanLike
-    | CanVote
+    | CanVoteInGeneral
+    | CanVoteOnIdea  -- can vote on one idea
+    | CanVoteOnTopic -- can vote on one topic
```

Comment:
`CanVoteInGeneral` is a super-type of the other two.  I wonder if there is a way to avoid that?